spec/features/user_searches_voters_spec.rb

```diff
@@ -0,0 +1,53 @@
+require 'rails_helper'
+
+feature 'user searches voters', %(
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.